gnmi_server/server_test.go

```diff
@@ -158,7 +159,7 @@ func getRedisClient(t *testing.T) *redis.Client {
 	dbn := spb.Target_value["COUNTERS_DB"]
```

Comment:
>Target_value [](start = 12, length = 12)

Should we remove enum Target? Everything should come from json config.

Reply:
I didn't remove it. Instead, updating the map values with the data read from database_config.json file when do init() in proto pkg. After that the we register use the same name and struct.

Reply:
I suggest follow the design of c++ or python. something like `sdcfg.GetDbId("COUNTERS_DB")`

---
In reply to: [344860708](https://github.com/Azure/sonic-telemetry/pull/24#discussion_r344860708) [](ancestors = 344860708)

Reply:
added GetDbId()